README.md

```diff
@@ -39,7 +39,7 @@ Actions are triggered by GitHub platform events directly in a repo and run on-de
 
 - [Official Site](https://github.com/features/actions)
 - [Official Documentation](https://help.github.com/en/actions)
-- [Official Actions organization](https://github.com/actions)
```

Comment:
I don't want to remove this, it's an important link. The Other two are ok as additions in subcategories.

Reply:
Reverted!